src/components/global/componentFunctions.js

```diff
@@ -20,8 +22,10 @@ function getIcon(icon) {
     cross: CrossIcon,
     play: PlayIcon,
     pause: PauseIcon,
+    logout: LogoutIcon,
     reset: ResetIcon,
     lightMode: DarkModeIcon,
+    darkMode: LightModeIcon,
```

Comment:
This should be `lightMode: LightModeIcon` and `darkMode: DarkModeIcon`. At the moment the wrong icon is displaying (i.e. shows icon to enable dark mode while already in dark mode).

Reply:
Ah yep, that makes sense, thanks for clearing that up!